src/index.ts

```diff
@@ -521,6 +521,26 @@ export const types17: RegistryTypes = {
   },
 };
 
+export const types18: RegistryTypes = {
+  ...types17,
+  // DID management update
+  DidCreationDetails: {
```

Comment:
bc the name is different now we wouldn't overwrite but add an additional type. I guess you'd need to get rid of the old ones? Either set them to undefined, or you can remove them like so:
```typescript
const { DidCreationOperation, DidDeletionOperation, DidUpdateOperation, ...selectedTypes17 } = types17
```

Reply:
https://github.com/KILTprotocol/type-definitions/pull/12/commits/454cde640fe5d1931bfa0bff5aac4907cb3fd390?

Reply:
sure! did you test whether it works? I'm wondering because this keeps the keys in the resulting object, with value undefined. in the worst case we could JSON stringify and parse to get rid of the keys.

Reply:
if the registry parses it like this I'm happy to merge it as is

Reply:
I guess the only way to test these is via the SDK?